doc/guides/security-release-process.md

```diff
@@ -58,9 +58,19 @@ information described.
 * [ ] Request releaser(s) to start integrating the PRs to be released.
 
 * [ ] Notify [docker-node][] of upcoming security release date: ***LINK***
+  ```text
+  Heads up of Node.js security releases Month Day Year
+
+  As per the Node.js security release process this is the FYI that there is going to be a security release Day Month Year
```

Comment:
There's an inconsistency in date formats: "Month Day Year" vs "Day Month Year"

I'm from a part of the world that uses "Day Month Year" but I'm more concerned with being unambiguous (probably okay if we use words for "Month" and not numbers) and consistent.

Reply:
I've updated this to be `Day Month Year` now.